.prettierignore

```diff
@@ -4,3 +4,4 @@ package.json
 dist
 .percy-debug
 vendors
+.yarn
```

Comment:
😆

Reply:
Good call. Added that in our internal repos too. :)

Reply:
You need a pretty fast computer to run prettier on those files 😄